.github/stale.yaml

```diff
@@ -0,0 +1,19 @@
+name: "Stale issue handler"
+on:
+  workflow_dispatch:
+  schedule:
+  - cron: "0 0 * * *"
+
+jobs:
+  stale:
+    runs-on: ubuntu-latest
+    steps:
+    - uses: actions/stale@v3
+      with:
+        repo-token: ${{ secrets.GITHUB_TOKEN }}
+        stale-issue-message: 'This issue is stale because it has been open 30 days with no activity. Remove stale label / comment or this will be closed in 5 days'
+        stale-pr-message: 'This PR is stale because it has been open for 30 days with no activity. Remove stale label / comment or this will be closed in 5 days'
+        days-before-stale: 30
+        days-before-close: 5
+        stale-pr-label: 'no-pr-activity'
+        stale-issue-label: 'no-issue-activity'
```

Comment:
add a newline at the end (recommended to configure your editor to do it automatically on save)